src/ui/simulator/toolbox/components/datagrid/renderer/connection.h

```diff
@@ -137,9 +138,11 @@ class connectionNTCdirect : public connectionNTC
 public:
     connectionNTCdirect(wxWindow* parent, Toolbox::InputSelector::Connections* notifier);
     ~connectionNTCdirect() override = default;
+    IRenderer::CellStyle cellStyle(int col, int row) const override;
 
 private:
     void setMatrix(Data::AreaLink* link) override;
+    Antares::Matrix<>* parameters;
```

Comment:
You should make **parameters** a private variable in parent class **connectionNTC**.
You should change **parameters** into **parameters_** or **pParameters**

Reply:
You mean protected ? If private, child classes `connectionNTCindirect` and `connectionNTCdirect` can't access it.

Reply:
Yes, I mean **protected**, sorry